index.js

```diff
@@ -8,19 +8,22 @@ const list = ['New York', 'Los Angeles', 'Chicago', 'Houston', 'Phoenix', 'Phila
 const searchBar = document.getElementById('typeahead');
 const suggestionDisplay = document.getElementById('suggestions');
 
-var suggestions = [];
-var firstOption;
+let suggestions;
+let HTMLSuggestions;
+let firstOption;
 
 searchBar.addEventListener('input', (event) => {
     const input = event.target.value;
-    if(input.length > 0) {
+    if (input.length > 0) {
         const regex = new RegExp(`^${input}`, 'i');
-        suggestions = list.sort().filter(v => regex.test(v));
+        suggestions = statesList.sort().filter(word => regex.test(word));
     } else {
         suggestions = [];
     }
     displaySuggestions();
-    firstOption = document.querySelector('li');
+    HTMLSuggestions = Array.from(document.querySelectorAll('li.suggestion'));
```

Comment:
1. I would recommend taking a look at getElementsByClassName instead of querySelectorAll? Looks like you already have a class name going!

2. `HTMLSuggestion` sounds more robotic, can we give this a contexual name?

Reply:
Can you use the variables `firstOption` & `lastOption` values on demand inside the necessary functions instead of having it as a global variable? This eliminates the unwanted dependency on the values between different functions.